test/Base_Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -686,6 +686,34 @@ add_specs suite_builder =
             meta_type = Meta.meta Standard.Base.Network
             meta_type.catch . should_equal (Private_Access.Error Nothing Nothing 'meta' 'No Meta.meta for private objects')
 
+        group_builder.specify "function with argument" <|
+            plus x = x+1
+
+            case Meta.meta plus of
+                fn : Meta.Function ->
+                    fn.is_resolved . should_be_true
+                unknown -> Test.fail "Expecting Meta.Function, but was: "+unknown.to_text
+
+        group_builder.specify "function without an argument" <|
+            means = 42
```

Comment:
I don't think `means = 42` is "function without an argument". I think that you would need to change it like this:
```
means =
    42
```
in order for it to be a function. `means = 42` is a local variable binding.